### ISSUE TITLE
Sonobuoy fails to start when you have Completed pods in kube-system

### ISSUE THREAD

**Issue by bismarck** [heptio/sonobuoy#83]
Sonobuoy fails start when you have Completed pods (i.e. those created by Jobs) in the kube-system namespace.  Here's the error I'm getting

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/e2e.go:221
Sep 28 15:23:55.754: Error waiting for all pods to be running and ready: 1 / 100 pods in namespace "kube-system" are NOT in RUNNING and READY state in 10m0s
POD                                   NODE                          PHASE     GRACE CONDITIONS
elastic-delete-index-1506556860-nskhn ip-10-203-21-240.ec2.internal Succeeded       [{Initialized True 0001-01-01 00:00:00 +0000 UTC 2017-09-28 00:01:00 +0000 UTC PodCompleted } {Ready False 0001-01-01 00:00:00 +0000 UTC 2017-09-28 00:01:05 +0000 UTC PodCompleted } {PodScheduled True 0001-01-01 00:00:00 +0000 UTC 2017-09-28 00:01:00 +0000 UTC  }]

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/e2e.go:172
```

If I remove the Completed pods, Sonobuoy starts without an issue.

**Comment by ncdc** [comment]
@timothysc related to this maybe? https://github.com/kubernetes/kubernetes/pull/53222

**Comment by timothysc** [comment]
This looks more like an upstream bug in the e2e suite.

**Comment by ncdc** [comment]
Yeah that's what 53222 is about

**Comment by timothysc** [comment]
@bismark do you have the full e2e.log?

**Comment by timothysc** [comment]
k, the upstream issue is the root... I'll trace through there, but we may not get a fix until 1.8.1.  
Closing this in preference of upstream PR.
